src/polyfill.js

```diff
@@ -40,6 +39,8 @@ function setup() {
         sanitizeFor(localName, input) {
         // TODO: should parse/sanitize/filter/validate values for localName.
           const context = document.createElement(localName);
+          // Q: What is the expected return value of _fragmentParser? It looks like the expected return type might be a DocumentFragment
```

Comment:
Oh, Right!?
AFAICT, you could
-  create a Range
- `selectNode` on the context element (created above)
-  `createContextualFragment` on thati nput string

I guess that's what `_fragmentParser` should do that on the inside.

Reply:
Oh, awesome! I guess we can delete everything in` _fragmentParser` except for [the following code block](https://github.com/0xedward/sanitizer-polyfill/pull/1/files#diff-d64aaabdffa3cd9703c8597141f114de6a2cd97f9cb8a2fda3bc09aa1fbd148bR68-R71) then :)